src/styles/mobile.scss

```diff
@@ -155,6 +155,11 @@
     }
   }
 
+  .blog-container .recent-posts {
+    /* Place article before recent posts lists on mobile */
+    order: 1;
```

Comment:
Would prefer this to be inside `article-reader.scss` 👀 

As we want to soon move to CSS modules

Reply:
Fair point, can do!